apps/fipsinstall.c

```diff
@@ -140,7 +140,8 @@ static int do_mac(EVP_MAC_CTX *ctx, unsigned char *tmp, BIO *in,
     return ret;
 }
 
-static int load_fips_prov_and_run_self_test(const char *prov_name)
+static int load_fips_prov_and_run_self_test(const char *prov_name,
+                                            int *is30fipsprov)
```

Comment:
`is140_2` would be a better name here since that's the underlying difference.

Reply:
ping @slontis to respond to the above comment